apps/remix-ide/src/app/ui/landing-page/landing-page.js

```diff
@@ -240,53 +240,43 @@ export class LandingPage extends ViewPlugin {
       })
     }
 
-    const startSolidity = () => {
-      this.appManager.ensureActivated('solidity')
-      this.appManager.ensureActivated('udapp')
-      this.appManager.ensureActivated('solidityStaticAnalysis')
-      this.appManager.ensureActivated('solidityUnitTesting')
+
+    const startSolidity = async () => {
+      await this.appManager.activatePlugin(['solidity', 'udapp', 'solidityStaticAnalysis', 'solidityUnitTesting'])
       this.verticalIcons.select('solidity')
     }
-    /*
-    const startWorkshop = () => {
-      this.appManager.ensureActivated('box')
-      this.appManager.ensureActivated('solidity')
-      this.appManager.ensureActivated('solidityUnitTesting')
-      this.appManager.ensureActivated('workshops')
-      this.verticalIcons.select('workshops')
-    }
-    */
-
     const startPipeline = () => {
-      this.appManager.ensureActivated('solidity')
-      this.appManager.ensureActivated('pipeline')
-      this.appManager.ensureActivated('udapp')
+      this.appManager.activatePlugin(['solidity', 'pipeline', 'udapp'])
     }
-    const startDebugger = () => {
-      this.appManager.ensureActivated('debugger')
+    const startDebugger = async () => {
+      await this.appManager.activatePlugin('debugger')
       this.verticalIcons.select('debugger')
     }
-    const startMythX = () => {
-      this.appManager.ensureActivated('solidity')
-      this.appManager.ensureActivated('mythx')
+    const startMythX = async () => {
+      await this.appManager.activatePlugin(['solidity', 'mythx'])
       this.verticalIcons.select('mythx')
     }
-    const startSourceVerify = () => {
-      this.appManager.ensureActivated('solidity')
-      this.appManager.ensureActivated('source-verification')
+    const startSourceVerify = async () => {
+      await this.appManager.activatePlugin(['solidity', 'source-verification'])
       this.verticalIcons.select('source-verification')
     }
-    const startPluginManager = () => {
-      this.appManager.ensureActivated('pluginManager')
+    const startPluginManager = async () => {
+      await this.appManager.activatePlugin('pluginManager')
       this.verticalIcons.select('pluginManager')
     }
+    /*
```

Comment:
remove

Reply:
I don't know, it was already commented. Should I ? ^^

Reply:
ah, sorry you moved it? I guess you can. it's an old workshop plugin.